packages/rs-dpp/src/data_contract/document_type/class_methods/try_from_schema/v0/mod.rs

```diff
@@ -130,13 +133,28 @@ impl DocumentTypeV0 {
                 .unwrap_or(default_mutability);
 
         // Extract the properties
-        let property_values =
-            Value::inner_optional_btree_map(schema_map, property_names::PROPERTIES)?
-                .unwrap_or_default();
+        let property_values = Value::inner_optional_index_map::<u64>(
+            schema_map,
+            property_names::PROPERTIES,
+            property_names::POSITION,
+        )?
+        .unwrap_or_default();
+
+        #[cfg(feature = "validation")]
+        if validate {
```

Comment:
We need to validate that position starts form 0 and there is no gaps

Reply:
It's done already for gaps. But good point on starting at 0.

Reply:
No actually it also already verifies it starts at 0.

Reply:
You're right, my bad